### DISCUSSION TITLE
Love it and hate it, the new API Explorer

### DISCUSSION THREAD

**Discussion by Nice2Work** [quasarframework/quasar#9009] [Ideas / Proposals] [answered]
Guys, the new API explorer is really great BUT, if you want to see the example, you have to have another tab open, which would be JUST A LINK to the original example !! So yes, love it, hate it 😁 
I am never good in explaining, but for example you want to see the QIMG properties, you see it all, but NOT a link to https://quasar.dev/vue-components/img#introduction, which in many cases makes sense if you are deciding what to use 😊👍

**Comment by rstoenescu** [comment, marked as answer]
You'll only have reasons to love it after the next docs deployment. Also, thanks for donating to Quasar!

![explorer](https://user-images.githubusercontent.com/4665641/115940710-3fb32500-a4ab-11eb-8b9e-85ffd64e57ab.png)

**Reply by Nice2Work** [reply]
Awesome, I knew you guys are the best, I would put money into your stock 😁👍😉
